### ISSUE TITLE
Sample webpack config

### ISSUE THREAD

**Issue by isaacplmann** [synapse-wireless-labs/component-lab#11]
Can you post a sample webpack config just to get this running with an angular cli app?  I'd like to get this running, but it's not worth writing a whole webpack config just for the component lab.

**Comment by isaacplmann** [comment]
Here's what I ended up doing.

In `component-lab.config.js`:
```ts
const NgCliWebpackConfig = require('@angular/cli/models/webpack-config').NgCliWebpackConfig;

module.exports = {
  /**
   * Webpack configuration object used to load your experiments
   */
  webpackConfig: new NgCliWebpackConfig({
    target: 'development',
  }).config,
  ...
}
```

And in the lab modules, make sure to import `polyfills.ts`:

```ts
import 'path/to/polyfills.ts';
```

This might be helpful to add to the README.md.  I'll leave this open so you can decide if you want to add this to the readme.

**Comment by brandonroberts** [comment]
Fixed via #24
